sql/payday.sql

```diff
@@ -76,12 +76,27 @@ CREATE INDEX ON payday_payment_instructions (participant);
 CREATE INDEX ON payday_payment_instructions (team);
 ALTER TABLE payday_payment_instructions ADD COLUMN is_funded boolean;
 
+UPDATE payday_payment_instructions ppi
+   SET due = s.due
+  FROM (SELECT participant, team, SUM(due) AS due
```

Comment:
Well, I'm thinking of a situation like this:

1. Payday 0 - subscription amount $3, uncharged, resulting in due $3
2. Subscription amount changed to $4 i.e new record entered with amount = $4 (and due = 0)
3. Payday 1 - ```current_payment_instructions``` would pick up the latest record, with due = 0 and the existing due of $3 would be missed

In this situation, we'd need some way to transfer the existing due to the new record at step 2. Thought of using an AFTER INSERT trigger, but updating ```payday_payment_instructions``` seemed simpler.